packages/aiqtoolkit_opentelemetry/src/aiq/plugins/opentelemetry/register.py

```diff
@@ -0,0 +1,214 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+from pydantic import Field
+
+from aiq.builder.builder import Builder
+from aiq.cli.register_workflow import register_telemetry_exporter
+from aiq.data_models.telemetry_exporter import TelemetryExporterBaseConfig
+
+logger = logging.getLogger(__name__)
+
+
+class LangfuseTelemetryExporter(TelemetryExporterBaseConfig, name="langfuse"):
+    """A telemetry exporter to transmit traces to externally hosted langfuse service."""
+
+    endpoint: str = Field(description="The langfuse OTEL endpoint (/api/public/otel/v1/traces)")
+    public_key: str = Field(description="The Langfuse public key", default="")
+    secret_key: str = Field(description="The Langfuse secret key", default="")
+    resource_attributes: dict[str, str] = Field(default_factory=dict,
+                                                description="The resource attributes to add to the span")
+
+    # Batch size control options
+    batch_size: int = Field(default=100, description="The batch size for the telemetry exporter.")
+    flush_interval: float = Field(default=5.0, description="The flush interval for the telemetry exporter.")
+    max_queue_size: int = Field(default=1000, description="The maximum queue size for the telemetry exporter.")
+    drop_on_overflow: bool = Field(default=False, description="Whether to drop on overflow for the telemetry exporter.")
+    shutdown_timeout: float = Field(default=10.0, description="The shutdown timeout for the telemetry exporter.")
```

Comment:
These properties are repeated. Lets have inheritance in the configs or use a mixin